includes/admin/load.php

```diff
@@ -86,9 +96,6 @@ function perflab_admin_pointer( $hook_suffix ) {
 		return;
 	}
 
-	$current_user = get_current_user_id();
-	$dismissed    = explode( ',', (string) get_user_meta( $current_user, 'dismissed_wp_pointers', true ) );
-
 	if ( in_array( 'perflab-admin-pointer', $dismissed, true ) ) {
 		return;
 	}
```

Comment:
Move this check after line no 81 because the update-related condition is checked on all pages, so it is not needed if the pointer is already disabled.

Reply:
Edit thanks :)